Sources/CircularProgress/CircularProgress.swift

```diff
@@ -9,6 +9,7 @@ public final class CircularProgress: NSView {
 	private var finishedObserver: NSKeyValueObservation?
 	private var cancelledObserver: NSKeyValueObservation?
 	private var indeterminateObserver: NSKeyValueObservation?
+	internal var hasMouseEntered:Bool = false/*Required for onOver / onOut to work*/
```

Comment:
This should be `private`.

Reply:
I think its internal because an extension needs access to it. extensions can't access private's vars.

Reply:
✅ Although it will become internal again when we refactor into using more extensions :D